app/src/main/kotlin/com/hippo/ehviewer/ui/settings/AdvancedScreen.kt

```diff
@@ -142,6 +144,13 @@ fun AdvancedScreen(navigator: DestinationsNavigator) {
                 title = stringResource(id = R.string.preload_thumb_aggressively),
                 value = Settings::preloadThumbAggressively,
             )
+            var animateItems by Settings.animateItems.asMutableState()
```

Comment:
This should not be available on API 29 above

Reply:
Done.